tests/stress/compute_flow_with_threads.py

```diff
@@ -261,17 +242,45 @@
     # Retrieve algorithm output and log files
     output = None
     for i in range(len(status["results"])):
-        result = None
         result_type = status["results"][i]["type"]
         print(f"Fetch result index {i}, type: {result_type}")
-        result = ocean.compute.result(DATA_asset, job_id, i, bob_wallet)
+        result = ocean.compute.result(
+            DATA_asset, compute_service, job_id, i, consumer_wallet
+        )
         assert result, "result retrieval unsuccessful"
+        print(f"result index: {i}, type: {result_type}, contents: {result}")
 
         # Extract algorithm output
         if result_type == "output":
             output = result
     assert output, "algorithm output not found"
+    unpickle_result(output)
+
 
-    # Unpickle the gaussian model result
-    model = pickle.loads(output)
+def unpickle_result(pickled):
+    """Unpickle the gaussian model result"""
+    model = pickle.loads(pickled)
     assert len(model) > 0, "unpickle result unsuccessful"
+
+
+def concurrent_c2d(concurrent_flows: int, repetitions: int):
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+    mint_fake_OCEAN(config)
+    with ThreadPoolExecutor(max_workers=concurrent_flows) as executor:
+        for _ in range(concurrent_flows * repetitions):
+            executor.submit(
+                c2d_flow_readme,
+                ocean,
+                "brainin",
+                "https://raw.githubusercontent.com/oceanprotocol/c2d-examples/main/branin_and_gpr/branin.arff",
+                "gpr",
+                "https://raw.githubusercontent.com/oceanprotocol/c2d-examples/main/branin_and_gpr/gpr.py",
+                "python-brain",
+            )
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(["concurrent_flows", "repetitions"], [(1, 2), (3, 1), (5, 5)])
```

Comment:
I have decreased the number of repetitions & number of workers because the ether balance is not enough for all the tests from this folder. When these tests will be run, at some point `barge` needs to be pruned (for filling the accounts again with ETH).
After 2 runs of `consume_flow_with_threads.py` with 5 workers and 5 repetitions
![Screenshot from 2022-04-19 13-17-46](https://user-images.githubusercontent.com/50501033/163983249-45b337ae-d354-4a37-9f1e-7d341d7cfd5a.png)

Reply:
I think these tests need to demonstrate at least 20 concurrent users. So I think we need to send more ETH to the wallets.

Reply:
From where? From other ganache wallets? It works fine one time with 20 workers, but it won't run for all scenarios + plus the other tests.

Reply:
I think we should increase the default wallet balance in barge to 1 million ETH.

```console
ganache-cli -e 1000000
```

Reply:
@mariacarmina I created barge PR to fix this.
https://github.com/oceanprotocol/barge/pull/286